utils/python/CIME/SystemTests/system_tests_common.py

```diff
@@ -48,6 +48,19 @@ def __init__(self, case, expected=None):
         self._case.set_value("TEST",True)
         self._case.flush()
 
+    def fail_test(self):
```

Comment:
Better encapsulation of runstatus management. We don't want classes outside of this one directly accessing this member.